extension/src/popup/ducks/accountServices.ts

```diff
@@ -447,6 +447,7 @@ const authSlice = createSlice({
           applicationState || APPLICATION_STATE.MNEMONIC_PHRASE_CONFIRMED,
         publicKey,
         allAccounts,
+        error: "",
```

Comment:
@piyalbasu I'm resetting the auth error if it comes back fulfilled, I think that's fine but just making sure (noticed it when logging in and entering the /verify-account view directly)

Reply:
Yup, nice